openpype/hosts/flame/plugins/publish/collect_test_selection.py

```diff
@@ -16,6 +16,7 @@ class CollectTestSelection(pyblish.api.ContextPlugin):
     order = pyblish.api.CollectorOrder
     label = "test selection"
     hosts = ["flame"]
+    active = False
```

Comment:
Attribute `enabled` would be better for collector.

Reply:
not available in `pyblish.api.ContextPlugin`